archetypes/authors/_index.md

```diff
@@ -20,52 +22,18 @@ organizations:
   address: 
   # Department
   department: 
-  # Job title
-  job_title: 
-
-# Short bio (displayed in user profile at end of posts)
-bio: 
-
-# Person's interests
-interests:
-- 
-# Person's education
-education:
-  courses:
-  - course: 
-    institution: 
-    year: 
 
 # Academic Networking
 accounts:
   # Forum name on OHDSI forum
-  ohdsi_forum_name: {{ .Name }}
+  ohdsi_forum_name: 
+  # Forum profile URL
+  ohdsi_forum_profile_url:
```

Comment:
What is the difference between these two?
`ohdsi_forum_name: daniel_prieto`
`ohdsi_forum_profile_url: https://forums.ohdsi.org/u/daniel_prieto`
Now the website generates `ohdsi_forum_profile_url` from `ohdsi_forum_name`